pkg/services/ngalert/state/manager.go

```diff
@@ -66,9 +66,9 @@ func NewManager(logger log.Logger, metrics *metrics.State, externalURL *url.URL,
 	return manager
 }
 
-func (st *Manager) Close(ctx context.Context) {
+func (st *Manager) Close() {
```

Comment:
I would keep this as it was before

```suggestion
func (st *Manager) Close(ctx context.Context) {
```